### DISCUSSION TITLE
What is the preferred way to set sysctl variables?

### DISCUSSION THREAD

**Discussion by deviantintegral** [abiosoft/colima#233] [Q&A] [answered]
I would like to increase the max file watchers as I am working with large projects that have more than 8192 files: https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc

While I can do this manually every time I start colima, ideally this happens automatically. I thought I could add `provision` to `colima.yml`, but it's removed when I run `colima start`.

**Comment by abiosoft** [comment, marked as answer]
For now, your best option is to use the `provision` section of a Lima override config.
Nonetheless, this should be included in Colima by default and will look at adding it in future releases.

> While I can do this manually every time I start colima, ideally this happens automatically. I thought I could add provision to colima.yml, but it's removed when I run colima start.

Yeah, this is not yet supported as proper configuration file support is in the works.
Considering the colima primarily generates a Lima config, I do not want to deviate entirely away from that.
